tests/db/utils_test.py

```diff
@@ -0,0 +1,59 @@
+# encoding: utf-8
+# Copyright (C) 2017 Lucas Hoffmann
+# This file is released under the GNU GPL, version 3 or a later revision.
+# For further details see the COPYING file
+from __future__ import absolute_import
+
+import email
+import unittest
+
+from alot.db import utils
+
+
+class TestGetParams(unittest.TestCase):
+
+    mailstring = '\n'.join([
+        'From: me',
+        'To: you',
+        'Subject: header field capitalisation',
+        'Content-type: text/plain; charset=utf-8',
+        'X-Header: param=one; and=two; or=three',
+        "X-Quoted: param=utf-8''%C3%9Cmlaut; second=plain%C3%9C",
+        'X-UPPERCASE: PARAM1=ONE; PARAM2=TWO'
+        '\n',
+        'content'
+        ])
```

Comment:
It's not a huge deal, but why using `str.join` instead of a  triple quote string?